kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -9,22 +9,25 @@ ssui_root="$repos_root/manageiq-ui-self_service"
 
 rm -rf $app_root
 
-git clone <%= @appliance_checkout.remote %> $appliance_root
+git clone <%= "--separate-git-dir /tmp/appliance.git" if @build_type == "release" %> \
+        <%= @appliance_checkout.remote %> $appliance_root
```

Comment:
Good point @bdunne.  It looks like it's been asked about a few times. It was asked about [here](http://talk.manageiq.org/t/miq-appliance-upgrade/755/12), [here](http://talk.manageiq.org/t/upgrading-from-darga-n-to-darga-n-1/1574), and finally using `git` and a followup to use export/import db into new appliance was advised [here](http://talk.manageiq.org/t/upgrade-to-darga/1520/6).  Either way, we should have a clear way to do upgrades... if git isn't used for upgrades, then, I'd be ok with removing the `.git` after we announce it.